rotkehlchen/chain/ethereum/names.py

```diff
@@ -171,20 +182,25 @@ def global_addressbook_address_to_name(
     return db_addressbook.get_addressbook_entry_name(
         book_type=AddressbookType.GLOBAL,
         address=address,
+        blockchain=blockchain,
     )
 
 
 def hardcoded_address_to_name(
         _: DBHandler,
         address: ChecksumEvmAddress,
+        blockchain: SupportedBlockchain,
 ) -> Optional[str]:
     """Returns the name of a known address or None if there is no such address"""
+    if blockchain != SupportedBlockchain.ETHEREUM:
+        return None
```

Comment:
We shoudl move those to the global db at some point. Maybe this release is fine and we would be able to get rid of this piece of code. But not this PR